README.md

```diff
@@ -11,11 +11,95 @@ Methods and tools for simulating SPI data.
 First create a conda environment with the required dependencies using the `enviroment.yml` file as follows:
 
     conda env create --file environment.yml
+or
+
+    conda env create -f environment.yml
+
+
+If the conda enivronment already exists but new packages have been added to environment.yml then update it using
+
+    conda env update --prefix ./env --file environment.yml  --prune
+Activate the environment using
+
+    conda activate simSPI
 
 Then download:
 
     git clone https://github.com/compSPI/ioSPI.git
 
+
+
+# Linear Simulator
+There are two types of simulator in this package:
+Linear and TEM.
+
+The linear simulator uses the linear model of the Cryo-EM physics to generate data.
+This involves fist rotating the biomolecule and obtaining its 2D tomographic projection.
+This 2D projection is then modulated with a CTF (Contrast Transfer Function) in fourier domain (convolution with corresponding PSF in image domain) and then shifted using scpecified shift values.
```

Comment:
shifted in real or fourier?

Reply:
I would write `then shifted in the Fourier domain using scpecified shift values.`